src/app/gui/inputs/service.js

```diff
@@ -4,6 +4,8 @@ const {t} = require('core/i18n/i18n.service');
 function Service(options = {}) {
   // set state of input
   this.state = options.state || {};
+  // in case is set default input value to input
+  this.set_input_default_value = false;
```

Comment:
@volterra79 is this a setter? maybe not, `has_default_value` or `has_default` would be more self-explanatory?

Reply:
@Raruto rename with has_default_value 7011a4a9b63a20eddd6cccd850742f490e213fad